changelog/fragments/ansible-drain-time.yaml

```diff
@@ -0,0 +1,5 @@
+entries:
+  - description: >
+      For Ansible-based operators, allow passing a --drain-time to configure the duration the manager should wait before stopping.
```

Comment:
Why call this --drain-time? Does this terminology come from somewhere?

Reply:
No real reason or preference here. Any suggestions for what to name it?

Reply:
`--graceful-shutdown-timeout`

Reply:
Updated all references!